static/spec/openapi.json

```diff
@@ -549,6 +551,19 @@
             }
           }
         }
+      },
+      "ErrorResponse": {
+        "type": "object",
+        "properties": {
+          "message": {
+            "type": "string",
+            "example": "Something went wrong"
+          },
+          "code": {
+            "type": "string",
+            "example": "123"
+          }
```

Comment:
The code type should be int.  https://github.com/RedHatInsights/export-service-go/blob/main/exports/jsonerror_response.go#L18

Reply:
Changed!